### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by steff456**
This PR is part of https://github.com/conda-incubator/jupyterlab-conda-store/issues/13

This PR,

- [x] Adds an issue template for bug reports
- [x] Adds an issue template for feature requests
- [x] Adds an issue template for general issues

## Notes

- Given that we don't have labels in the repo yet there's some `TODOs` that we may need to resolve before merging this PR.
- I don't have powers in the repo to create the labels so I'll wait either for powers to do them or for someone to create them.

### CODE REVIEW

**Review by kcpevey — CHANGES_REQUESTED** [review]
On hold until labels are added to this repo

**Comment by kcpevey** [comment]
Blocked until https://github.com/conda-incubator/conda-store/pull/521 is merged and the changes are made here as well.

**Comment by trallard** [comment]
This is now unblocked

**Comment by steff456** [comment]
This PR needs an approval for merging! I already copied all the changes from [the conda-store PR](https://github.com/conda-incubator/conda-store/pull/521)

File: .github/ISSUE_TEMPLATE/config.yml
Comment (steff456) [inline .github/ISSUE_TEMPLATE/config.yml:3]: From my understanding, there are shared docs for the JupyterLab extension and the conda-store-ui

**Review by pavithraes — APPROVED** [review]
@steff456 Thanks! The CI is failing on the linter again, could you please verify/update that? I'll approve so you can merge when ready. :)

**Comment by trallard** [comment]
@steff456  can you fix the linting issues?

**Comment by steff456** [comment]
This PR needs an approval for merging :)!

**Pull request merged**
